src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -435,15 +437,14 @@ function get_sysinfo_item_html($itemkey) {
 		$catsort = $catdata['catsort'];
 		$data[$catsort]['cattitle'] = $catdata['cattitle']; // add key for category and get category title
 		$data[$catsort]['itemstoshow'][$itemsort] = $itemdata;  // create a key for item and add static item data
-		$data[$catsort]['itemstoshow'][$itemsort]['item_html'] = '<tr>' . $item_html . '<tr>';
+		$data[$catsort]['itemstoshow'][$itemsort]['item_html'] = '<tr>' . $item_html . '</tr>';
 	}
 }
 if (count($data) == 0) {
 	// adds a single item containing the special key for the "no sysinfo selected" item if nothing else will display
-	$data[0] = array(
-		'cattitle' => '',
-		'itemstoshow' => array('itemsort' => '', 'item_html' => '<tr>' . get_sysinfo_item_html('ALL_HIDDEN') . '<tr>')
-		);
+	$data[0]['cattitle'] = '';
+	$data[0]['itemstoshow'][0]['itemtitle'] = '';
+	$data[0]['itemstoshow'][0]['item_html'] = '<tr>' . get_sysinfo_item_html('ALL_HIDDEN') . '</tr>';
```

Comment:
This creates the proper data structure to match what is expected below. The other array method above did not work with the code below. This needs to be fixed in the master PR also.